File: Java Programming/Competitive Programming/Coding Challenges 2/program5.java
/*
5. Write a program to check whether a number is positive, negative, or zero.
Input: 
Output:
*/

class Logic
{
    void checkSign(int iNo)
    {
        if (iNo < 0 ) 
        {
          System.out.println("Number is Negative : "+iNo); 
        }
        else if ( iNo > 0 ) 
        {
         System.out.println("Number is Positive : "+iNo);    
        }
        else
        {
            System.out.println("Number is Zero : "+iNo);  
        }
    }
}

class program5 
{
    public static void main(String[] args) 
    {
        Logic obj=new Logic();
        obj.checkSign(-8);
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 6/program5.cpp
//5.Accept number from user and check whether number is even or odd.
//Output:  Enter Number : 11
//         Number is odd
//              &
//         Enter Number : 2
//         Number is Even

#include <iostream>
using namespace std;

bool ChkEven(int iNo)
{
    //Logic
    if ((iNo % 2) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    int iValue = 0;
    bool bRet = false;

    cout << "Enter Number : ";
    cin >> iValue;

    bRet = ChkEven(iValue);

    //Display result
    if (bRet == true)
    {
        cout << "Number is Even";
    }
    else
    {
        cout << "Number is odd";
    }

    return 0;
}